test/WebJobs.Script.Tests/Workers/Rpc/RpcWorkerProcessTests.cs

```diff
@@ -163,6 +164,18 @@ public void ErrorMessageQueue_Full_Enqueue_Success()
             Assert.Equal("Error2,Error3,Error4", exceptionMessage);
         }
 
+        [Fact]
+        public void VerifySanitizedErrorMessage_Success()
```

Comment:
Is this test doing same thing as the `SanitizeString` test in SanitizerTests.cs? I am wondering this joined string can be passed as an inline data to that test.

Reply:
Yes its doing the same thing. I will remove this one. The second input data I added to sanitizeString is this joined string.